modulefiles/module_gwci.orion.lua

```diff
@@ -0,0 +1,20 @@
+help([[
+Load environment to run GFS workflow ci scripts on Orion
+]])
+
+-- Temporary until official hpc-stack is updated
+
+prepend_path("MODULEPATH", "/apps/modulefiles/core")
```

Comment:
Should be using the Orion hpc-stack just like Hera:
```suggestion
prepend_path("MODULEPATH", "/apps/contrib/NCEP/hpc-stack/libs/hpc-stack/modulefiles/stack")

load(pathJoin("hpc", "1.1.0"))
load(pathJoin("hpc-intel", "2018.4"))
load(pathJoin("hpc-impi", "2018.4"))
```